src/qt/splashscreen.cpp

```diff
@@ -137,7 +137,7 @@ SplashScreen::~SplashScreen()
 bool SplashScreen::eventFilter(QObject * obj, QEvent * ev) {
     if (ev->type() == QEvent::KeyPress) {
         QKeyEvent *keyEvent = static_cast<QKeyEvent *>(ev);
-        if(keyEvent->text()[0] == 'q') {
+        if (keyEvent->key() == Qt::Key_Q) {
```

Comment:
I think this slightly changes the behavior from accepting 'q' to 'q' and 'Q' (shift-q). But I'm fine with this, personally.